FusionMCPBridge/handlers/design/geometry_impl2.py

```
@@ -0,0 +1,642 @@
+        elif Plane == "YZ":
+            sketch = sketches.add(rootComp.yZConstructionPlane)
+        
+        for i in range(len(points)-1):
+            start = adsk.core.Point3D.create(points[i][0], points[i][1], 0)
+            end = adsk.core.Point3D.create(points[i+1][0], points[i+1][1], 0)
+            sketch.sketchCurves.sketchLines.addByTwoPoints(start, end)
+        sketch.sketchCurves.sketchLines.addByTwoPoints(
+            adsk.core.Point3D.create(points[-1][0], points[-1][1], 0),
+            adsk.core.Point3D.create(points[0][0], points[0][1], 0)
+        )
+    except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)